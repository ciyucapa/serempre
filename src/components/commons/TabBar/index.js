import React from 'react';
import {Link} from "gatsby"

import {Assets} from '../../../resources';

const TabBar = () => {
    return (
        <div style={{backgroundColor: '#1f1e1e', display: 'flex', flexDirection: 'row', justifyContent: 'center', alignItems: 'center', width: '100%', height: 50, position: 'fixed', bottom: 0, paddingRight: 5, zIndex: 5}}>
            <div style={{width: 200, height: '100%', display: 'flex', flexDirection: 'row', justifyContent: 'space-between', alignItems: 'center', backgroundColor: '#FFFFFF'}}>
                <Link to="/" style={{padding: 10}}>
                    <img src={Assets.images.list} style={{width: 30, height: 30}} />
                </Link>
                <Link to="/maps" style={{padding: 10}}>
                    <img src={Assets.images.map} style={{width: 30, height: 30}} />
                </Link>
            </div>
        </div>
    );
};

export default TabBar;
