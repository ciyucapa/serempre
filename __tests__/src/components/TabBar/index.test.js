import React from 'react';
import renderer from 'react-test-renderer';

import TabBar from '../../../../src/components/commons/TabBar';

describe('TabBar', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<TabBar />)
            .toJSON();

        expect(tree).toMatchSnapshot();
    });
});
