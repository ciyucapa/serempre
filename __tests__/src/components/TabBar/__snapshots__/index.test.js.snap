// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabBar renders correctly 1`] = `
<div
  className="sc-bdnylx bCbvVP"
>
  <div
    className="sc-gtssRu iwvBCG"
  >
    <a
      href="/"
    >
      <img
        className="sc-dlnjPT cjOmng"
        src="/static/images/list.png"
      />
    </a>
    <a
      href="/maps"
    >
      <img
        className="sc-dlnjPT cjOmng"
        src="/static/images/map.png"
      />
    </a>
  </div>
</div>
`;
